### PR TITLE
Change GitHub source configuration procedure

### PR DESCRIPTION
**Pull request by astefanski-sumo**
Change GitHub source configuration procedure to disable automatic timestamp parsing and use receipt time as _messageTime

img/integrations/app-develpoment/Field_GitHub.png referenced in this md file needs to be updated in line with this pull request. I could not find this file in repo to include replacement in this pull request. I'll provide the file on slack. 

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by astefanski-sumo** [comment]
As discussed on #talk-apps: https://sumologic.slack.com/archives/C1PB1DMB3/p1678105949649269?thread_ts=1677659130.433209&cid=C1PB1DMB3

**Comment by astefanski-sumo** [comment]
As I can see I can share image here. Below is replacment for Field_GitHub.png. 

Field_GitHub

**Review by JKashyap96 — APPROVED** [review]
LGTM

File: docs/integrations/app-development/github.md [hunk 1]
```diff
@@ -124,8 +124,9 @@ In this step, you configure a Hosted Collector to receive Webhook Events from Gi
 2. Configure an[ HTTP Source](/docs/send-data/hosted-collectors/http-source/logs-metrics) on the Hosted Collector.
     * For Source Category, enter any string to tag the output collected from this Source, such as **GitHub**.
     * Click **+Add Field** and provide the following:
-        * **Field Name.** `_convertHeadersToFields`
-        * **Value.** true
+        * **Field Name:** `_convertHeadersToFields`
+        * **Value:** `true`
+    * Expand **Advanced Options for Logs (Optional)** section, then ***uncheck (disable)*** option **Extract timestamp information from log file entries** in **Timestamp Parsing**
```
Comment (kimsauce) [inline docs/integrations/app-development/github.md:129]:
```suggestion
        * **Field Name**. `_convertHeadersToFields`
        * **Value**. `true`
    * Expand **Advanced Options for Logs (Optional)** section, then **uncheck (disable)** option, then **Extract timestamp information from log file entries** in **Timestamp Parsing** 
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Minor style fix

**Review by stacykor — APPROVED** [review]

**Comment by stacykor** [comment]
Builds. Just fails spell check.

**Review by stacykor — APPROVED** [review]

**Pull request merged**